src/components/Subheading/tests/Subheading.test.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import {mount} from 'enzyme';
+import Subheading from '../Subheading';
+
+describe('Subheading Component Suit', () => {
+    describe('Test children property', () => {
+        it('should verify children property is defined', () => {
+            const subheadingWrapper = mount(<Subheading>Online store dashboard</Subheading>);
+            expect(subheadingWrapper.prop('children')).toBe('Online store dashboard');
+        });
+
+         it('should verify children property is not defined', () => {
+            const subheadingWrapper = mount(<Subheading/>);
+            expect(subheadingWrapper.prop('children')).toBeUndefined();
+        });
+    });
+
+    describe('Test element property', () => {
+        it('should verify element property when set as h1', () => {
+            const subheadingWrapper = mount(<Subheading element="h1">Online store dashboard</Subheading>);
+            expect(subheadingWrapper.prop('element')).toBe('h1');
+        });
+
+         it('should verify element property when set as h2', () => {
+            const subheadingWrapper = mount(<Subheading element="h2">Online store dashboard</Subheading>);
+            expect(subheadingWrapper.prop('element')).toBe('h2');
```

Comment:
As discussed, you can add specific test for markup when we set element property as h2.